solutions.txt

```diff
@@ -105,7 +100,7 @@ Exercise 9:
 */
 
 // Complete Exercise 9 below...
-let allFoods = foods.concat['->'];
+var allFoods = ('taco -> sushi -> cupcake -> tofu -> cheeseburger')
```

Comment:
hmmm.. here, you're doing what's called 'hard-coding'. Ideally, you would do something that solves the problem *programmatically*, meaning that it would work for any array of foods you gave it, not just for this one.

try using the [`join()` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray/join). the third example in the sandbox is the closest to what we're trying to do here